src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const daysForMinDiscount = 3;
+  const daysForMaxDiscount = 7;
+  const costPerDay = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+  const totalCost = days * costPerDay;
+
+  switch (true) {
+    case (days < daysForMinDiscount) :
+      return totalCost;
+
+    case (days >= daysForMinDiscount && days < daysForMaxDiscount) :
+      return totalCost - minDiscount;
+
+    case (days >= daysForMaxDiscount) :
+      return totalCost - maxDiscount;
+  }
```

Comment:
It works, but switch/case structure should not be used this way.
Rewrite with if/return.